packages/vue/src/components/ReactiveBase/index.jsx

```diff
@@ -62,6 +64,7 @@ const ReactiveBase = {
 				this.$props.theme,
 			),
 			store: this.store,
+			$analytics: computed(()=>this.analyticsRef),
```

Comment:
@SavvyShah What are you doing here with `computed`?

Reply:
The properties in provide are not reactive by default. So I followed [this guide.](https://vuejs.org/guide/components/provide-inject.html#working-with-reactivity)